### PR TITLE
Provision AMI with Mesos 1.8.0

### PR DESCRIPTION
**Pull request by kensipe**
Summary:
In order to do seccomp we need to run against Mesos 1.8.0. This patch also updates the AMI
for master.

JIRA issues: DCOS_OSS-5272

Co-Authored-By: Karsten Jeschkies <kjeschkies@mesosphere.io>
Co-Authored-By: Andreas Neumann <aneumann@mesosphere.io>

### CODE REVIEW

**Comment by kensipe** [comment]
issue with mesos on this version of OS
```
[31mWARN [0;39m[18:10:56 LaunchQueueIntegrationTest-MesosMaster-32824] mesos: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.27' not found (required by mesos)
[31mWARN [0;39m[18:10:56 LaunchQueueIntegrationTest-MesosAgent-32825] mesos: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.27' not found (required by mesos)
```

**Comment by kensipe** [comment]
Failing because Mesos 1.8 is installed correctly on Debian 9
blocked by https://jira.mesosphere.com/browse/DCOS_OSS-5096

File: ami/AMI.md [hunk 1]
```diff
@@ -4,12 +4,19 @@ Marathon Jenkins AMI
...
+Mesos version used by marathon: https://github.com/mesosphere/marathon/blob/b00f71136a7e35cb76c7df136d49b16b9ead2689/project/Dependencies.scala#L128
 
 Here is an example on how to install Mesos 1.5.0 using maws (https://github.com/mesosphere/maws#osx--macos) which will result in image named 'JenkinsMarathonCI-Debian8-2018-02-02'
 
 ```bash
 $(maws login "Team 10")
 AWS_PROFILE=273854932432_Mesosphere-PowerUser packer build -color -var "ami_name=JenkinsMarathonCI-Debian9-$(date +%Y-%m-%d)" -var 'mesos_version=1.5.0-2.0.1' marathon-jenkins-ami.json
 ```
+
+Here is an example on how to install Mesos 1.8.0-rc using maws (https://github.com/mesosphere/maws#osx--macos) which will result in image named 'JenkinsMarathonCI-Debian9-2019-04-22'
```
Comment (jeschkies) [inline ami/AMI.md]:
```suggestion
Here is an example on how to install Mesos 1.8.0-rc using [maws](https://github.com/mesosphere/maws#osx--macos) which will result in image named 'JenkinsMarathonCI-Debian9-2019-04-22'
```

**Comment by kensipe** [comment]
Till/Benno and team will provide a new release of mesos for debian with seccomp

**Comment by jeschkies** [comment]
Somehow our subnet to build the AMI was removed. I filed a ticket https://jira.mesosphere.com/browse/DCOS-54243.

File: ami/install.bash [hunk 2]
```diff
@@ -42,68 +59,48 @@ apt-get install -y \
...
 wget https://www.python.org/ftp/python/3.6.2/Python-3.6.2.tgz
 tar xvf Python-3.6.2.tgz && cd Python-3.6.2/
-./configure --enable-optimizations
+./configure
 make -j
 sudo make install
 cd ../ && rm -r Python-3.6.2
+# Use this instead of the manual python compile when we switch to buster base image. Make sure we get Python3.6
+#echo -e "\n=== Install Python 3, Pip and Flake8 ==="
+#apt-get install python3-distutils python3 -y
```
Comment (jeschkies) [inline ami/install.bash]: I think these can go.

Comment (ANeumann82) [reply ami/install.bash]: I've added them again so I don't have to figure out what packages to install when we switch to buster...

**Comment by jeschkies** [comment]
What's the state of this @kensipe and @ANeumann82?

File: tests/integration/src/test/scala/mesosphere/marathon/integration/MesosAppIntegrationTest.scala [hunk 3]
```diff
@@ -140,8 +139,8 @@ class MesosAppIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonT
             endpoints = Seq(raml.Endpoint(name = "task1", hostPort = Some(0))),
             healthCheck = Some(MesosHttpHealthCheck(portIndex = Some(PortReference("task1")), path = Some("/ping"))),
             volumeMounts = Seq(
-              VolumeMount(Some("python"), s"$containerDir/python", false),
-              VolumeMount(Some("data"), s"$containerDir/data", true)
+              VolumeMount(Some("python"), s"$containerDir/python", true),
+              VolumeMount(Some("data"), s"$containerDir/data", false)
```
Comment (jeschkies) [inline tests/integration/src/test/scala/mesosphere/marathon/integration/MesosAppIntegrationTest.scala]: @meichstedt and @timcharper, `true` marks the volume as read-only. Why did the tests pass on Mesos 1.7 and earlier? They should have always failed.

**Pull request merged**